### PR TITLE
Add `Dioxus-cli` Document

### PR DESCRIPTION
**Pull request by mrxiaozhuox**

### CODE REVIEW

**Comment by mrxiaozhuox** [comment]
@jkelleyrtp Please help me to check the grammar haha !

**Comment by BarryYan** [comment]
@mrxiaozhuox Why not merge it into [Cli](https://github.com/dioxusLabs/cli) repo? :smiley:

File: docs/cli/src/configure.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+1. ***style*** - include some style(CSS) file into html.
+   ```
+   style = [
+      # include from public_dir.
+      "./assets/style.css",
+      # or some asset from online cdn.
+      "https://cdn.jsdelivr.net/npm/bootstrap/dist/css/bootstrap.css"
+   ]
+   ```
+2. ***script*** - include soe script(JS) file into html.
```
Comment (BarryYan) [inline docs/cli/src/configure.md:88]: `soe` -> `some`

**Comment by mrxiaozhuox** [comment]
> @mrxiaozhuox Why not merge it into [Cli](https://github.com/dioxusLabs/cli) repo? 😃

Because I think we need put all docs in `dioxus` repo.

**Comment by mrxiaozhuox** [comment]
😃

File: docs/cli/src/cmd/build.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Build
+
+`dioxsu build` command can help you `pack & build` a dioxus project.
```
Comment (jkelleyrtp) [inline docs/cli/src/cmd/build.md:3]: dioxus

File: docs/cli/src/introduction.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Introduction
+
+📦✨ **Dioxus-Cli** is a tool to help get dioxus projects off the ground.
+
+![dioxus-logo](https://dioxuslabs.com/guide/images/dioxuslogo_full.png)
+
+It include `dev server`, `hot reload` and some `quick command` to help you use dioxus.
```
Comment (jkelleyrtp) [inline docs/cli/src/introduction.md:7]: include should be plural (includes)

**Review by jkelleyrtp — CHANGES_REQUESTED** [review]
Just some simple grammar stuff (plurals, capitalization)

**Comment by jkelleyrtp** [comment]
Awesome - thank you!

We'll probably make some more tweaks to docs before releasing the next version.

**Pull request merged**